beta/src/components/MDX/TeamMember.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ */
+
+import * as React from 'react';
+import Image from 'next/image';
+import {IconTwitter} from '../Icon/IconTwitter';
+import {IconGitHub} from '../Icon/IconGitHub';
+import {ExternalLink} from '../ExternalLink';
+import {IconNewPage} from 'components/Icon/IconNewPage';
+import {H3} from './Heading';
+import {IconLink} from 'components/Icon/IconLink';
+
+interface TeamMemberProps {
+  name: string;
+  title: string;
+  permalink: string;
+  children: React.ReactNode;
+  photo: string;
+  twitter?: string;
+  github?: string;
+  personal?: string;
+}
+
+// TODO: good alt text for images/links
+export function TeamMember({
+  name,
+  title,
+  permalink,
+  children,
+  photo,
+  github,
+  twitter,
+  personal,
+}: TeamMemberProps) {
+  if (name == null || title == null || permalink == null || children == null) {
+    throw new Error(
+      'Expected name, title, permalink, and children for ' + name ??
+        title ??
+        permalink ??
+        'unknown'
+    );
+  }
+  return (
+    <div className="pb-6 sm:pb-10">
+      <div className="flex flex-col sm:flex-row height-auto">
```

Comment:
```suggestion
      <div className="flex flex-col sm:flex-row height-auto items-center">
```
It looks better when image is at the center? What do you think 😅 

| before | after |
| -------| ----- | 
| <img width="880" alt="Screenshot 2022-12-08 at 2 38 03 PM" src="https://user-images.githubusercontent.com/32865581/206405284-cadc6ad8-c331-4f07-9823-5cdc99621af3.png"> |  <img width="897" alt="Screenshot 2022-12-08 at 2 38 21 PM" src="https://user-images.githubusercontent.com/32865581/206405374-fa16f056-ea1f-47ee-b329-3ddeb00bdf16.png"> |

Reply:
Thanks but I prefer the current design.